cscs-checks/microbenchmarks/gpu/kernel_latency/kernel_latency.py

```diff
@@ -65,8 +65,8 @@ class gpu_kernel_latency_check(GpuKernelLatency):
     def set_valid_prog_environs(self):
         cs = self.current_system.name
         if cs in {'dom', 'daint'}:
```

Comment:
`hooks.py` (line 28) loads `cudatoolkit/11.1.0_3.39-4.1__g484e319`.
This is not needed with `PrgEnv-nvidia`

Reply:
@jgphpc Well, for some cases it is.

```bash
[     FAIL ] (5/5) CudaSamplesTest_conjugateGradientCudaGraphs on dom:gpu using PrgEnv-nvidia
```
Loading that version of the cudatoolkit would solve this.

Reply:
The cuda graphs check now passes and the loading of that specific cudatoolkit version is being removed in #2026